src/modules/endpoint/sagas/__tests__/loadCityContent.spec.js

```diff
@@ -102,6 +76,7 @@ describe('loadCityContent', () => {
 
     expect(await new AppSettings().loadSelectedCity()).toBe('augsburg')
     expect(await dataContainer.getLastUpdate(city, language)).toBe(lastUpdate)
+    expect(await RNFetchBlob.fs.readFile(new DatabaseConnector().getMetaCitiesPath(), '')).toMatchSnapshot()
```

Comment:
Replace this with a more abstract expect. `loadCityContent` is too far away from persistence to expect on actual files.
You could expect on `new DatabaseConnector().loadLastUsages()`.
Also it's beneficial to avoid snapshots and inline the expectation.